aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -13,6 +13,7 @@
 #ifndef AT_PER_OPERATOR_HEADERS
 #include <ATen/Functions.h>
 #else
+#include <ATen/ops/_propagate_xla_data.h>
```

Comment:
@bdhirsh Added the corresponding `_propagate_xla_data` change here. Please review the 9f6d3391f67038a9f5c21ad675d34238e5ca53b3 commit.